docs/styles/theme.css

```diff
@@ -90,12 +91,15 @@ div[style="overflow: auto"]{
   top: 36px;
   width: 100%;
   padding: 0 !important;
+  background-color: var(--background);
 }
 .search .input-wrap {
-  margin-left: 10px;
+  margin-left: 5px;
+  padding: 5px 0;
 }
 .search .results-panel.show {
   padding: 10px;
+  border-bottom: 4px dashed var(--accent);
```

Comment:
```suggestion
border-bottom: 1px solid var(--accent);
```

This looks more decent, WDYT?
We can add bottom `box-shadow` as well!

Reply:
sure let me do the changes

Reply:
![image](https://user-images.githubusercontent.com/96580571/220973456-caa4f483-3215-46e2-b039-a1f7bba94ff4.png)
@vedant-jain03 is this enough?

Reply:
LGTM!